scripts/install

```diff
@@ -70,6 +82,8 @@ function init_venv()
     pip install Flask
     pip install Flask-Script
     pip install Flask-SQLAlchemy
+    #psycopg2 driver for sqlalchemy postgresql dialect (either that or pg8000)
+    if [ "$db_type" = 'postgresql' ]; then pip install psycopg2; fi
```

Comment:
Uh don't you need `[[ ]]` and `==` ? ( I never know @.@ )

Reply:
Seems like 
> [ ] are bash Builtins (POSIX compliant)
> [[ ]] are bash Keywords (not so POSIX compliant)

Kind of didn't know the difference either before I [look for it](https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash). Which seems best ?